Gem/Code/Source/RHI/TextureMapExampleComponent.cpp

```diff
@@ -636,12 +636,12 @@ namespace AtomSampleViewer
         int vertices = 0;
         for (int i = 0; i < arraySize; i += 6)
         {
-            indexBuffer[i + 0] = vertices + 0;
-            indexBuffer[i + 1] = vertices + 3;
-            indexBuffer[i + 2] = vertices + 1;
-            indexBuffer[i + 3] = vertices + 1;
-            indexBuffer[i + 4] = vertices + 3;
-            indexBuffer[i + 5] = vertices + 2;
+            indexBuffer[i + 0] = static_cast<uint16_t>(vertices + 0);
```

Comment:
`vertices` can be declared as a `uint16_t` here (and rename it to `indexCursor` or something please while you're here 😉)